docs/CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * Support for multiline text. ([#965])
 * Support for logging to file, toggle for logging to stdout. ([#976], [#994])
 * Added `shadow_update()` and `shadow_fixed_update()` to the `State` trait. ([#1006])
-
+* Added configurable width for debug lines ([#1016])
```

Comment:
You need to also add a link definition below

Reply:
Done